File: Pattern15.py

#       *
#      ***
#     *****
#    *******
#   *********

n = int(input("Enter the row number:"))
for i in range(1,n+1):
    for k in range(i,n+1):
        print(" ",end='')
    for j in range(1,2*i):
        print("*",end='')
    print()

File: Pattern15.go
//       *
//      ***
//     *****
//    *******
//   *********

package main

import "fmt"

func main(){

	var n int
	fmt.Print("Enter the row number:")
	fmt.Scanf("%d", &n)

	for i := 1; i <= n; i++ {
		for k := i; k <= n; k++ {
			fmt.Print(" ")
		}
		for j := 1; j <= 2*i-1; j++ {
			fmt.Print("*")
		}
		fmt.Println()
	}
}